js/src/lib/deeplink.js

```diff
@@ -0,0 +1,30 @@
+export function handleTabs() {
```

Comment:
What's the advantage of putting this in a separate file, given it's tightly-coupled to the page? I'd suggest just putting this in `js/src/install.js` directly